### PR TITLE
add top 10 records in dropdown if more than 100 records are present

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Bugfix
- Enhancement / new feature
### Description
- fix - Typeahead filter - gives misleading information
- show the first 10 results in dropdown with an indication that there are more available

### CODE REVIEW

**Comment by kornys** [comment]
@enmasse-ci run tests testcase=FirefoxConsoleTest

File: console/console-init/ui/src/pages/AddressSpaceDetail/AddressList/AddressListFilter.tsx [hunk 1]
```diff
@@ -156,11 +167,21 @@ export const AddressListFilter: React.FunctionComponent<IAddressListFilterProps>
...
     ) {
       //To display dropdown if fetched records are less than 100 in count.
-      if (response.data.addresses.total > 100) {
-        setNameOptions([]);
+      if (
+        response.data.addresses.total >
+        MAX_ITEM_TO_DISPLAY_IN_TYPEAHEAD_DROPDOWN
+      ) {
+        let list = [];
+        for (let i = 0; i < 10; i++) {
```
Comment (anukritijha) [inline console/console-init/ui/src/pages/AddressSpaceDetail/AddressList/AddressListFilter.tsx]: I think it's better to store 10 in a const variable in a global file and import it from there, as it's being used in multiple files.

Comment (anukritijha) [reply console/console-init/ui/src/pages/AddressSpaceDetail/AddressList/AddressListFilter.tsx]: Also, this logic here is repetitive and we can have a common function which just accepts a list of records and push the 10 records in an array

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=FirefoxConsoleTest

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests profile=shared testcase=web.**

**Pull request merged**